README.md

```diff
@@ -18,26 +18,37 @@ that they remain accurate and complete.
 Before running `linguist`, make sure:
 
 - You have GitHub SSH authentication set up.
-- Your SSH key is added to GitHub.
+- **If your SSH private key is _not_ passphrase-protected**, you only need to
```

Comment:
Since users no longer need to run `ssh-add` manually from the CLI, I think this part can be removed from the README.